pillar/components/system.sls

```diff
@@ -19,3 +19,9 @@ system:
   ntp:
     time_server: time.seagate.com
     timezone: UTC             #To be replaced with input
+  service_user:
+    name: cortxub
+    password:
```

Comment:
are you deliberately not using "secret" to avoid password encryption?

Reply:
As per my understanding only fields with "secret" are encrypted in lyveutil.encrypt module, @ypise can confirm.

Reply:
>are you deliberately not using "secret" to avoid password encryption?
That block of data is related to usual system user params. I guess it's ok to in encryption logic. You may check that: https://github.com/Seagate/cortx-prvsnr/blob/e6d15babe156505f810b912ccd0918112cc42788/srv/_modules/pillar_ops.py#L53

Reply:
I see, thanks!